src/lib/src/calendar.component.ts

```diff
@@ -342,7 +342,7 @@ export class CalendarComponent implements OnInit, AfterViewInit, AfterContentChe
             });
             elem[0].dispatchEvent(widgetEvent);
         };
-        this.options.resourceRender = (resourceObj: any, labelTds: any, bodyTds: any) => {
+        (<any>this.options).resourceRender = (resourceObj: any, labelTds: any, bodyTds: any) => {
```

Comment:
@Jamaks This is clearly ugly
We somehow need to extend `OptionsInput` from `fullcalendar` to include the types from `fullcalendar-scheduler`